### ISSUE TITLE
Pane Layout with expanded panes on load

### ISSUE THREAD

**Issue by KaiSoellch** [siemens/ix#1108]
### Prerequisites

- [X] I have not leaked any internal company information.
- [X] I have not posted internal links or code snippets.

### What happened?

if a pane is expanded as default, it disappears

### What type of frontend frameware are you seeing the problem on?

Angular

### Which version of iX do you use?

v2.1.0

### Code to produce this issue.

```shell
https://stackblitz.com/edit/uy1kta?file=src%2Fapp%2Fpane-layout.ts,src%2Fapp%2Fpane-layout.html
```

**Comment by jul-lam** [comment]
Internal ref [IX-948]

**Comment by danielleroux** [comment]
A fix for this issue is now available in v2.1.1-beta.0.
We kindly ask you to install this version in your dev environment to check if it resolves the problem for you and give us some feedback about this.

**Comment by jul-lam** [comment]
I will close the issue. If anything is still not working, you can simply open a new issue with a reference to this one.
